docs-2.0/nebula-console.md

```diff
@@ -145,15 +186,15 @@ Fri, 20 Aug 2021 06:36:05 UTC
 Executed 3 times, (total time spent 3681/4734 us), (average time spent 1227/1578 us)
 ```
 
-## Sleep to wait
+### Sleep to wait
```

Comment:
forgive me, didn't get the meaning of sleep to wait. Is it better to say "Sleep and wait"?

Reply:
Thank you for bringing this up. Hadn't noticed that there was a problem here. Modified.